pkg/config/config_template.yaml

```diff
@@ -4464,6 +4464,8 @@ api_key:
     # probabilistic_sampler:
       ## @param sampling_percentage - number - optional - default: 100
       ## @env DD_OTLP_CONFIG_TRACES_PROBABILISTIC_SAMPLER_SAMPLING_PERCENTAGE - number - optional - default: 100
+      ## If `apm_config.probabilistic_sampler.enabled` is enabled, this config is ignored, `apm_config.probabilistic_sampler.enabled.sampling_percentage`
+      ## is used instead.
```

Comment:
This behavior has existed before, but it has now been explicitly documented. cc @ajgajg1134 

ProbabilisticSampler is not affected by the decision of OTLPReceiver’s probabilistic sampler.

https://github.com/DataDog/datadog-agent/blob/91794f654cad0d5a4ab1b455dfe626f30d8cb890/pkg/trace/agent/agent.go#L653-L665

https://github.com/DataDog/datadog-agent/blob/dcd0631cb285e0c18ac169d6ce593c5f1125d252/pkg/trace/sampler/probabilistic.go#L113-L168

Reply:
For info, we now have the `IsConfigured` function in the config to know if a setting was set by the user or comes from the defaults.